kiwi/system/root_bind.py

```diff
@@ -173,15 +173,7 @@ def move_to_root(self, elements):
 
         :rtype: list
         """
-        result = []
-        for element in elements:
-            normalized_element = os.path.normpath(element)
-            result.append(
-                normalized_element.replace(
-                    os.path.normpath(self.root_dir), os.sep
-                ).replace('{0}{0}'.format(os.sep), os.sep)
-            )
-        return result
+        return Path.move_to_root(self.root_dir, elements)
```

Comment:
@schaefi I left it here to keep the API as it was. I have seen that this root_bind method is only used within the package manager classes, thus it's got just few in the code. It wouldn't be a big effort to completely remove it and use `Path.move_to_root` instead. Do you think it makes sense? It doesn't bother me leaving it as it is now.

Reply:
I'm almost convinced it will not cause any breakage and also looks like a nice change. Call me picky but can we leave that code part untouched for the moment ? The reason is that we in theory agreed on no bigger refactoring and I saw myself violating that a bit when I did the swap changes ;) As we plan to roll out everything we did to SLES at the end of this sprint I think it would be good to have not so many changes at code parts that not urgently needed it. Thoughts ?

Reply:
Yep absolutely fine with me. I also like having this sort of changes in a separate PR.

Reply:
ok, so once the PR is updated we can merge it. I'll approve prior that change